span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/jaeger/JaegerSpanNormalizer.java

```diff
@@ -59,13 +64,19 @@ public class JaegerSpanNormalizer {
   public static final String OLD_JAEGER_SERVICENAME_KEY = "jaeger.servicename";
 
   private static final String SPAN_NORMALIZATION_TIME_METRIC = "span.normalization.time";
+  private static final String SPAN_REDACTED_ATTRIBUTES_COUNTER = "span.redacted.attributes";
+  private final ConcurrentMap<String, Counter> spanAttributesRedactedCounters =
```

Comment:
nit: Use `Map` instead of `ConcurrentMap`.

Reply:
I think it should be thread-safe since 2 stream threads are using it.

Reply:
okay